deploy/helm/sumologic/values.yaml

```
@@ -2976,11 +2976,9 @@ otelagent:
     podLabels: {}
     ## Add custom annotations only to otelagent daemonset.
     podAnnotations: {}
-    ## Memory Ballast size should be max 1/3 to 1/2 of memory.
-    memBallastSizeMib: "250"
     image:
       repository: "public.ecr.aws/sumologic/opentelemetry-collector"
-      tag: "0.22.0-sumo"
+      tag: "0.38.0-sumo"
```

```suggestion
      tag: "0.38.1-sumo"
```

(in process of building) :)